build/cpp/func/src/func_server.cc

```diff
@@ -76,6 +76,6 @@ void func_server::RunServer(
 int main(int argc, char **argv) {
   google::InitGoogleLogging(argv[0]);
   LOG(INFO) << "func_server main commenced";
-  func_server::RunServer(warble_functions::WarbleFunctions::GetFuncMap());
+  func_server::RunServer(warble_functions::func_map);
```

Comment:
Warble Functions was changed from a class with static methods to a set of functions existing in the warble_functions namespace. As such, GetFuncMap() was removed.

Reply:
Good, I think it meets the requirements for ephemeral